include/nleobs.h

```diff
@@ -78,4 +78,19 @@ typedef struct {
 #endif
 } nle_seeds_init_t;
 
+typedef struct nle_settings {
+    /*
+     *  Path to NetHack's game files.
+     */
+    char hackdir[4096];
+    char options[32768];
+    char wizkit[4096];
```

Comment:
don't need TERM?

Reply:
just saw that below we always set to "ansi" in get env. is it worth including it here instead?

Reply:
We kind of require "ansi" as the value ever since dropping the dependency on ncurses, so any other user-supplied value would break it I think.